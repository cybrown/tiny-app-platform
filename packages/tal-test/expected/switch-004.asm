main()
  entry:
    Literal           true
    Literal           true
    Intrinsic         INTRINSIC_EQUAL
    JumpTrue          if_true_1
    Jump              if_false_2
  if_true_1:
    Literal           "ok"
    Jump              if_end_3
  if_false_2:
    Literal           true
    Literal           false
    Intrinsic         INTRINSIC_EQUAL
    JumpTrue          if_true_4
    Jump              if_false_5
  if_true_4:
    Literal           "ko"
    Jump              if_end_6
  if_false_5:
    Literal           "default"
    Jump              if_end_6
  if_end_6:
    Jump              if_end_3
  if_end_3:
