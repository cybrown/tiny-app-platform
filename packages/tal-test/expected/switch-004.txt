Operator === not compatible with type boolean
Operator === not compatible with type boolean
string